docs: prettier home page for the documentation portal (#309)
allinbits/cosmos-cash: docs/How-To/README.md

# How-to guides

> The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources can be found in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder. 

Reviewer: ```suggestion
The [How-To](./) folder contains how-to guides for different aspects of the Cosmos project. Additional resources are in the [seed scripts](https://github.com/allinbits/cosmos-cash/tree/main/scripts/seeds) folder. 

```

Reviewer: "Cosmos project" do we mean a blockchain that is built using the Cosmos SDK or do we mean generally "Cosmos ecosystem"?
